src/clj/rems/application/model.clj

```diff
@@ -541,6 +542,18 @@
 
              {}))))
 
+(defn- get-blacklisted-users [application blacklisted?]
+  (let [all-members (application-util/applicant-and-members application)
+        all-resources (map :resource/ext-id (:application/resources application))]
+    (vec
+     (for [member all-members
+           resource all-resources
+           :when (blacklisted? (:userid member) resource)]
+       {:blacklist/user member :blacklist/resource {:resource/ext-id resource}}))))
```

Comment:
I would expect this function to group by user. Perhaps do that or rename the function to something like `get-blacklisted-user-resources`. The name does not imply that it returns the resources as well or that it returns one row per user-resource -pair.

Reply:
changed the function & field name to `blacklist`